website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -265,7 +278,9 @@ Now your pipeline can accept `--input=value` and `--output=value` as command-lin
 ## 3. PCollections {#pcollections}
 
 The <span class="language-java">[PCollection](https://beam.apache.org/releases/javadoc/{{< param release_latest >}}/index.html?org/apache/beam/sdk/values/PCollection.html)</span>
-<span class="language-py">`PCollection`</span> abstraction represents a
+<span class="language-py">`PCollection`</span>
+<span class="language-go">[PCollection](https://github.com/apache/beam/blob/master/sdks/go/pkg/beam/pcollection.go#L39)</span>
```

Comment:
In the future, when pkg.go.dev renders the documentation, should this be the URL? https://pkg.go.dev/github.com/apache/beam/sdks/go/pkg/beam#PCollection

Reply:
Correct! Until the license issue is resolved though, linking to the code with marginal precision is better than nothing.